games/static/games/proto.js

```diff
@@ -515,8 +523,17 @@ class Board {
   updateIndexOf(rocket) {
     rocket.index = rocket.index + this.directionUp();
   }
-  liftRocketAt(rocketLocation) {
-    return this.squareAt(rocketLocation).pickCharacterUp();
+  liftRocketAt(rocketLocation, rocketId) {
+    if (rocketLocation < 0) {
+      clearInterval(rocketId)
+      throw "invalid location. should be bigger than 0"
```

Comment:
i think this is not necessary, this Error never appears at the console and also down the method there is an if statement that takes care of what happens when rocket hits the top

Reply:
yes we dont really need this check (i gave it only as an example if a guard clause that makes sure only valid params are allowed to the function)
here, i dont think this index will ever be nonpositive